pysap/plugins/mri/reconstruct/fourier.py

```diff
@@ -140,9 +140,12 @@ def __init__(self, samples, shape):
         shape: tuple of int
             shape of the image (not necessarly a square matrix).
         """
+        self.samples = samples
+        if samples.min() < -0.5 and samples.max() >= 0.5:
```

Comment:
Shouldn't this be an `or` rather than an `and`?

Reply:
Yes it should be an or